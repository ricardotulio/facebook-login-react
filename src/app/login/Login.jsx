import React from 'react'
import {
  InputText,
  Password,
  Button,
} from '../common'

const Login = () => (
  <form>
    <InputText id="login" name="login" />
    <Password id="password" name="password" />
    <Button type="button">SIGN IN</Button>
  </form>
)

export default Login