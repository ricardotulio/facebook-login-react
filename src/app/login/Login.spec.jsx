import React from 'react'
import renderer from 'react-test-renderer'

import Login from './Login'

test('login form must be built correctly', () => {
  const login = renderer.create(<Login />)
  let tree = login.toJSON()

  expect(tree).toMatchSnapshot()
})