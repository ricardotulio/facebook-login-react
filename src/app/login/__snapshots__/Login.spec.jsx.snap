// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login form must be built correctly 1`] = `
<form>
  <input
    id="login"
    name="login"
    type="text"
  />
  <input
    id="password"
    name="password"
    type="password"
  />
  <button
    type="button"
  >
    SIGN IN
  </button>
</form>
`;
